internal/sink/sink.go

```diff
@@ -24,6 +24,11 @@ import (
 	"github.com/openmeterio/openmeter/internal/meter"
 	kafkametrics "github.com/openmeterio/openmeter/pkg/kafka/metrics"
 	kafkastats "github.com/openmeterio/openmeter/pkg/kafka/metrics/stats"
+	"github.com/openmeterio/openmeter/pkg/models"
+)
+
+const (
+	defaultOnFlushSuccessTimeout = 5 * time.Second
```

Comment:
Just for context: if we cannot publish the usage for a given organization in this period of time in OpenMeter Cloud, we will skip that data.

In this sense I would consider adding at least a metric on the amount of timeouts here.